weblate/static/styles/screenshots/screenshot_detail.css

```diff
@@ -4,6 +4,10 @@
  * SPDX-License-Identifier: GPL-3.0-or-later
  */
 
+ #sources-listing {
+  margin-bottom: 0px;
+ }
```

Comment:
This is not what I meant, see the screenshots I've just posted.

Reply:
Uh I see. Fixing it.